HucaresWF/src/HucaresWF/HucaresWF.csproj

```diff
@@ -66,6 +70,7 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
+    <None Include="packages.config" />
```

Comment:
File added to compile but not included with commit

Reply:
Added file to git